config/default.yaml

```diff
@@ -1,5 +1,7 @@
 # /!\ YOU SHOULD NOT UPDATE THIS FILE, USE production.yaml instead /!\ #
 
+allow_webadmin_config: true
```

Comment:
put it in below `peertube.check_latest_version` instead and add it to `production.yaml.example`

Prefer `webadmin.config.edition.allowed: boolean` instead. So we can (in the future) enable only some settings for example

Reply:
What do you think of:

```yaml
webadmin:
  configuration:
    allow_edits: true
```

or do you see the possibility for new fields in the future?


```yaml
webadmin:
  configuration:
    # i could this happening:
    viewing:
      allowed: true
    edit: 
      allowed: true
      # but what other fields can exist here?
```

Reply:
Sorry for my late replay

I'd prefer the second option, just in case :)

Reply:
Alright I changed it